java/openjdk/Portfile

```diff
@@ -441,6 +415,12 @@ if {${subport} in [list openjdk12 openjdk13 openjdk14 openjdk15]} {
     PortGroup    obsolete 1.0
     # Can replace with openjdk17 when it is added and openjdk16 is obsoleted
     replaced_by  openjdk16
+} elseif {${subport} eq "openjdk8-openj9-large-heap"} {
+    PortGroup    obsolete 1.0
+    replaced_by  openjdk8-openj9
+} elseif {${subport} eq "openjdk11-openj9-large-heap"} {
+    PortGroup    obsolete 1.0
+    replaced_by  openjdk11-openj9
```

Comment:
Wouldn’t it be better to include these above when the subport is first declared ? I am not sure I see the advantage to separating them out down here ? I also wonder if doing so wouldn’t help with some of the nitpick warnings?

either way, can you also please leave an empty line after each portgroup line.

Reply:
I've refactored the Portfile to only have a single declaration of `PortGroup obsolete 1.0` and I've inlined all the `replaced_by` declarations.

`port lint` reports 0 errors, and `port lint --nitpick` only warns about lines that 'seem to hardcode the version number', but we want a different version per 'subport' in this case.